proto/trace.proto

```diff
@@ -20,3 +20,8 @@ enum Event {
 	TiKvCoprExecuteDagRunner = 1004;
 	TiKvCoprExecuteBatchDagRunner = 1005;
 };
+
+message ResourceGroupTag {
```

Comment:
I think this should not be put in trace. It has nothing related to trace.

Reply:
Ok, I add a new file `resourcetag.proto` to put this.